### ISSUE TITLE
LocationDropdown component accessibility issue

### ISSUE THREAD

**Issue by 11-071985** [microsoft/fluentui#19372]
### Environment Information

- **Package version(s)**:   "@microsoft/azureportal-reactview": "^1.42.0",
- **Browser and OS versions**: (fill this out if relevant)

#### Actual behavior:

I am using LocationDropdown from "import { LocationDropdown } from '@microsoft/azureportal-reactview/LocationDropdown';".  It has accessibility issue where labels "recommended" and "other" are not associated with list-items, which is causing screen-reader to not announce the label while navigating list-items. 

#### Expected behavior:

Screen reader should read the group before reading list-items.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal
Products/sites affected: (if applicable)

https://us-prod.asyncgw.teams.microsoft.com/v1/objects/0-wus-d8-568da8322540cc57e8842d3aa3f3c751/views/imgo

**Comment by gouttierre** [comment]
@shipragupta21 - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [DropDown ](https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/dropdown) examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
